src/ui/units/wizard/components/Dialogs/DialogPlaceholder/DialogPlaceholder.tsx

```diff
@@ -391,30 +391,57 @@ class DialogPlaceholder extends React.PureComponent<Props, State> {
         const sortHasMeasures =
             visualizationId !== WizardVisualizationId.Area && sort.some(isMeasureField);
 
+        let showDisabledTooltip = false;
+        let disabledTooltipContent = null;
+
         const radioButtons = AXIS_MODE_RADIO_BUTTONS.map((option) => {
             if (option.value === SETTINGS.AXIS_MODE.CONTINUOUS) {
+                // Axis mode always depends on first item in placeholder
+                let disabled = !isAllAxisModesAvailable(firstField) || sortHasMeasures;
+
+                if (disabled) {
+                    showDisabledTooltip = true;
+                    disabledTooltipContent = 'label_axis-mode-unavailable-type';
```

Comment:
I think you need to either write the more generalized text in the tooltip, or separate the keys for different disable cases.
Now, in the case of sorting, there will be a text that does not explain the reason for the disabling.